bedevere/stage.py

```diff
@@ -222,6 +222,9 @@ class Blocker(enum.Enum):
             thanks = BORING_THANKS
         comment = ACK.format(greeting=thanks, core_devs=core_devs)
         await gh.post(issue["comments_url"], data={"body": comment})
+        reviewers_url = pr_url + '/requested_reviewers'
```

Comment:
Too bad that isn't in the webhook data somewhere. 😞

Reply:
Would it be better to extract `"/request_reviewers"` into a global variable?

Reply:
Eh, probably not. This was just a comment of me lamenting about this.